lib/events/events.go

```diff
@@ -489,3 +506,31 @@ func Error(err error) *string {
 	str := err.Error()
 	return &str
 }
+
+type noopLogger struct{}
+
+func NewNoopLogger() Logger {
```

Comment:
Just `const NoopLogger Logger = &noopLogger{}` ? No function call needed

Reply:
Unfortunately `noopLogger` isn't a constant. And making it variable seems bad to me.

Reply:
I think it's ok?  Go does that with http.DefaultHttpClient etc.

Reply:
Yes. (Or `type noopLogger int` and then `const NoopLogger Logger = noopLogger(0)` but I think that's uglier. :)